README.md

```diff
@@ -1,3 +1,20 @@
 # Pulsar Edit
 
-This is the website repo for Pulsar Edit. It contains our documenation and other information users might be looking for. Visit the [Website](https://pulsar-edit.github.io).
+This is the website repo for Pulsar Edit. It contains our documentation and other information users might be looking for. Visit the [Website](https://pulsar-edit.github.io).
+
+
+## Guide
+
+### Recommended Tools:
+- [Toolbox](https://containertoolbx.org/): We recommend using this tool to aide in development as it helps with containerization.
+  - On Linux, Toolbox should be available to install via your native package manager. Alternatively, Toolbox can also be installed by utilizing [Brew](https://brew.sh/).
+  - On MacOS, Brew is also available. However, if Brew is not an option we recommend downloading Toolbox directly from the [official website](https://containertoolbx.org/).
+  - Yarn (don't set this up just yet, follow the guide below)
+  
+### Setup
```

Comment:
For consistency, there should be an additional newline

Reply:
Well, this section will have to be changed. Brew will probably have to still be there, but will have to find alternative container methods for these. Just like I told @confused-Techie because I just assumed toolbox was availible elsewhere and it's not. Podman may be itself(what toolbox is based on) but I'm not entirely sure on messing with that and creating the custom images we need.